src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/CropImageTests.java

```diff
@@ -0,0 +1,58 @@
+package com.wikia.webdriver.testcases.mercurytests.curatedcontenttests;
+
+import com.wikia.webdriver.common.contentpatterns.MercuryPaths;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.LoginPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.curatedcontent.CuratedMainPagePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.curatedcontent.EditorHomePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.curatedcontent.curatededitorform.ItemFormPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.curatedcontent.imageupload.CroppingToolPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.curatedcontent.imageupload.SearchForImagePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.curatedcontent.imageupload.UploadImageModalComponentObject;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+/**
+ * @ownership Content X-Wing
+ */
+public class CropImageTests extends NewTestTemplate {
+
+    public static final String SEARCH_IMAGE_QUERY = "U";
+
+    @BeforeMethod(alwaysRun = true)
+    public void beforeMethod() {
+        new LoginPage(driver).get().logUserIn(Configuration.getCredentials().userNameStaff2,
+                Configuration.getCredentials().passwordStaff2);
+    }
+
+    @Test(groups = "CropImageTest_001")
+    @Execute(onWikia = "mercuryemptycceditor")
+    public void CropImageTest_001_cropOptionInModal() {
+        CuratedMainPagePageObject curatedMainPagePageObject = new CuratedMainPagePageObject(driver);
+
+        //@TODO this shouldn't be necessary
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)